frontend/components/atoms/_style.scss

```diff
@@ -14,6 +14,8 @@
 @import 'hamburger/hamburger';
 @import 'gutenberg/colors';
 @import 'gutenberg/editor';
+//@import 'gutenberg/core-blocks';
+//@import 'gutenberg/image';
```

Comment:
why did you commented this files?

Reply:
image seemed to be missing and core-blocks was conflicting by adding it's own styles to the core-blocks. This is just a PoC so don't worry about this - if we do go down this root we'll have to remove all SASS anyway.